docs/banzai/index.rst

```diff
@@ -2,80 +2,136 @@
-is only required for Sinsitros. This is currently done by the preprocessor.
+Again, only the Sinistro frames currently read out with multiple amplifiers so mosaicing the amplifiers
+is only required for Sinsitros.
 
 
 Trim
 ====
-After the overscan is subracted, the bias, dark, flat-field, and science images are trimmed
-based on the TRIMSEC header keyword.
+After the overscan is subtracted, the bias, dark, flat-field, and science images are trimmed
+based on the ``TRIMSEC`` header keyword.
+
+
+Bias Subtraction
+================
+Full frame bias images are subtracted from each of the darks, flat field images, and science frames.
+The master bias frame that was taken closest in time to the current data will be used.
+This will add a few counts of noise to each pixel, but it solves two problems. First, if there is systematic
+structure in the read out, this will be removed. Second, this will remove the bias values for images
+that do not have an overscan region.
+
+If no bias frame exists for this instrument, the data will not be reduced and an exception will be
+raised.
+
+
+Dark Subtraction
+================
+Full-frame master dark frames, scaled to the exposure time of the frame,
+are subtracted from all flat-field and science images. The most recent
+master dark frame is used. Often this is taken on the same day. If no dark frame exists for this
+instrument, the data will not be reduced and an exception will be raised.
+
+
+Flat Field Correction
+=====================
+Master flat field images (normalized to unity) are divided out of every science frame. The most recent
+master flat-field image for the given telescope, filter, and binning is used. If no flat field exists,
+the data will not be reduced and an exception will be raised.
+
+
+Source Detection
+================
+Source detection uses the "Source Extraction in Python" (SEP; https://github.com/kbarbary/sep).
+This is similar to Source Extractor, but is written purely in Python and Cython. This allows more
+customization.
+
+We estimate the background by taking a 3x3 median filter of the image and the doing a 32x32 block
+average of the image.
+
+We use the default match filter for source detection that is provided by SEP.
+
+We do aperture photometry using an elliptical aperture that is set by 2.5 times the Kron radius. This
+produces approximately the same results as ``FLUX_AUTO`` from SExtractor.
+
+We set the source detection limit at 3 times the global rms of the image. ``MINAREA`` is set to 5,
+the default. This should minimize false detections, but may miss the faintest sources.
+
+The catalog is returned as the ``CAT`` as fits binary table extension of the final science image. The catalog
+has the following columns: the position in pixel coordinates, (``X``, ``Y``), the flux (``Flux``),
+the error in the flux (``Fluxerr``), the semi-major and semi-minor axes (``a``, ``b``),
+and the position angle (``theta``).
+
```

Comment:
Is it worth a full catalog column list here?

Reply:
I've taken it out.